tests/test_repository.py

```
@@ -669,3 +669,107 @@ def test_clone_no_branch_when_empty(self, mock_repo: MagicMock) -> None:
+    @patch("jenkins_job_insight.repository.Repo")
+    def test_clone_into_with_token_injects_into_url(
+        self, mock_repo: MagicMock, tmp_path
+    ) -> None:
+        """clone_into with token injects it into the clone URL."""
+        manager = RepositoryManager()
+        target = tmp_path / "my-repo"
+        manager.clone_into(
+            "https://github.com/org/repo",
+            target,
+            depth=1,
+            token="tok123",  # pragma: allowlist secret
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to argument: "token"

(S106)